Bartosz/basics.js

```diff
@@ -0,0 +1,37 @@
+// 0.
+let apples = 20;
+let oranges = 30;
+console.log("apples = " + apples, "oranges = " + oranges); // apples = 20 oranges = 30
+
+// 1.
+console.log(apples == oranges); // false
+
+// 2.
+console.log(apples != oranges); // true
+
+// 3.
+console.log(apples > oranges); // false
+
+// 4.
+console.log(apples <= oranges); // true
+
+// 5.
+console.log(apples < oranges); // true
+
+// 6.
+let mangoes = 5;
+console.log(mangoes * apples > mangoes + oranges); // true
```

Comment:
console.log((mangoes * apples) > (mangoes + oranges));

same with 7, 9 and 10

Reply:
I though that something is wrong, I already commited and pushed the changes.